docs/manual/forms/cosign_flow.rst

```diff
@@ -0,0 +1,32 @@
+.. _manual_cosign_flow:
+
+Mede-ondertekenen
+=================
+
+Flow
+----
```

Comment:
```suggestion
Mede-ondertekenen
=================

Open Formulieren ondersteunt formulieren waarin een tweede persoon een inzending moet ondertekenen voor deze
verwerkt wordt, zoals een partner of voogd. Er is dus een extra persoon betrokken naast de invuller van het formulier.

Flow
----
```

I'd give a little introduction about what cosigning is and why the need could exist for that.